.buildkite/scripts/steps/integration-package.sh

```diff
@@ -3,4 +3,10 @@ set -euo pipefail
 
 source .buildkite/scripts/common.sh
 
-PACKAGES=tar.gz,zip,rpm,deb PLATFORMS=linux/amd64,linux/arm64,windows/amd64  SNAPSHOT=true EXTERNAL=true  mage package
+# Remove AGENT_PACKAGE_VERSION pinning as soon as 9.0.0 is released
+export AGENT_PACKAGE_VERSION=9.0.0
```

Comment:
same comment as in https://github.com/elastic/elastic-agent/pull/6750#discussion_r1948998820